### PR TITLE
zh-translation:docs/tasks/security/authorization/authz-ingress

### PR DESCRIPTION
**Pull request by my-git9**
zh-translation:docs/tasks/security/authorization/authz-ingress

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/security/authorization/authz-ingress/test.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```
Comment (zirain) [inline content/zh/docs/tasks/security/authorization/authz-ingress/test.sh:1]: @ericvn do we need copy the test to other language? I think those should verified in en version.

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+
+* 您可以使用在线代理服务使用不同的客户端 IP 访问入口网关，以验证请求是否被允许。
+
+* 如果您没有得到预期的响应，请查看应显示 RBAC 调试信息的入口网关日志：
+
+    {{< text bash >}}
+    $ kubectl get pods -n istio-system -o name -l istio=ingressgateway | sed 's|pod/||' | while read -r pod; do kubectl logs "$pod" -n istio-system; done
+    {{< /text >}}
+
+## 清理 {#Clean up}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:408]:
```suggestion
## 清理 {#clean-up}
```

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+        hpaSpec:
+          maxReplicas: 10
+          minReplicas: 5
+        serviceAnnotations:
+          service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+{{< /text >}}
+
+{{< /tip >}}
+
+## IP-based allow list and deny list
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:195]: 需要翻译。

File: content/zh/docs/tasks/security/authorization/authz-ingress/index.md [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+
+所有将流量引入 Kubernetes 的方法都涉及在所有工作节点上打开一个端口，实现这一点的主要功能是`NodePort`服务和`LoadBalancer`服务，甚至 Kubernetes 的`Ingress`资源也必须由 Ingress 控制器支持，该控制器将创建`NodePort`或`LoadBalancer`服务。
+
+* `NodePort` 只是在每个工作节点上打开一个 30000-32767 范围内的端口，并使用标签选择器来识别将流量发送到哪些 Pod。 您必须在工作节点前面手动创建某种负载均衡器或使用轮询模式的 DNS。
+
+* `LoadBalancer` 就像`NodePort` 一样，除了它还创建一个特定于环境的外部负载均衡器来处理将流量分配到工作节点。 例如，在 AWS EKS 中，`LoadBalancer` 服务将创建一个以您的工作程序节点为目标的经典 ELB。 如果您的 Kubernetes 环境没有 `LoadBalancer` 实现，那么它的行为就像 `NodePort`。 Istio 入口网关创建一个`LoadBalancer`服务。
+
+如果处理来自 `NodePort` 或 `LoadBalancer` 的流量的 Pod 没有在接收流量的工作节点上运行怎么办？ Kubernetes 有自己的内部代理，称为 kube-proxy，它接收数据包并将它们转发到正确的节点。
+
+## 原始客户端的源IP地址 {#Source IP address of the original client}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-ingress/index.md:59]:
```suggestion
## 原始客户端的源IP地址 {#source-ip-address-of-the-original-client}
```

**Comment by my-git9** [comment]
I can't understand

**Comment by zirain** [comment]
@ericvn can you take a look at this, `Kubernetes` and `Istio` are included by this .spell file.

**Comment by zirain** [comment]
@my-git9 I can test right now, but can you add `kubernetest` and `istio` into .spell file at root of this repo and run `make lint` to verify?

**Comment by my-git9** [comment]
> /ok-to-test

**Comment by my-git9** [comment]
> @my-git9 I can test right now, but can you add `kubernetest` and `istio` into .spell file at root of this repo and run `make lint` to verify?

Sorry, it's hard for me to get the gcr.io/istio-testing/build-tools:master-2022-03-24T20-56-29 image from my computer, I've tried many ways but still nothing success

**Comment by zirain** [comment]
> > @my-git9 I can test right now, but can you add `kubernetest` and `istio` into .spell file at root of this repo and run `make lint` to verify?
> 
> Sorry, it's hard for me to get the gcr.io/istio-testing/build-tools:master-2022-03-24T20-56-29 image from my computer, I've tried many ways but still nothing success

you can just add them in .spell file and push

**Review by zirain — APPROVED** [review]
lgtm

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**